odom_frame_id: odom #里程计坐标系
base_frame_id: base_link #底盘坐标系
map_frame_id: map #地图坐标系

mid_point_num: 3 #终点数量
mid_point_tolerance: [0.15, 0.15, 0.15] #终点容差 test1:0.2
mid_point_angle_tolerance: [3, 3, 3] #终点角度容差
mid_point_position: #终点位置
  [[0.250, -1.300, 0.0], [0.700, -1.30, 0.000], [1.200, -1.30, 0.000]]
mid_point_orientation: #终点姿态
  [[0.000, 0.000, 0.694, 0.720], [0.000, 0.000, 0.694, 0.720], [0.000, 0.000, 0.694, 0.720]]
Area_num: 3 #区域数量
Area_point1: #区域限制点1
  [[0.400, -1.0], [0.900, -1.0], [1.400, -1.0]]
Area_point2: #区域限制点2
  [[-0.100, -1.50], [0.400, -1.50], [0.900, -1.50]]